src/components/TransactionType.tsx

```diff
@@ -43,6 +43,23 @@ function getTypeIcon(type: string, color?: Color) {
   }
 }
 
+function getTypeTooltip(type: string): string {
+  switch (type) {
+    case "block_metadata_transaction":
```

Comment:
looks like its being used in different functions, wonder if it makes sense to extract it into an enum/type?

Reply:
yeah good call. let me add a TODO for it.